data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -302,26 +314,42 @@ def _create_downloadable_data(
                 ).astype("Int64")
                 df[column] = df_int
 
-            elif column in constants.FEMA_ROUND_NUM_COLUMNS:
+            elif column_type_dict[column] == "loss_rate_percentage":
                 # Convert loss rates by multiplying by 100 (they are percents)
                 # and then rounding appropriately.
                 df_100 = df[column] * 100
                 df[column] = floor_series(
                     series=df_100.astype(float64),
-                    number_of_decimals=constants.TILES_FEMA_ROUND_NUM_DECIMALS,
+                    number_of_decimals=config_object["rounding_num"][
+                        "loss_rate_percentage"
+                    ],
                 )
 
-            else:
-                # Round all other floats.
+            elif column_type_dict[column] == "float":
+                # Round the floats.
                 df[column] = floor_series(
                     series=df[column].astype(float64),
-                    number_of_decimals=constants.TILES_ROUND_NUM_DECIMALS,
+                    number_of_decimals=config_object["rounding_num"]["float"],
                 )
 
-        # sort by tract id
-        df_sorted = df.sort_values(self.GEOID_TRACT_FIELD_NAME)
+        # rename fields
+        column_rename_dict = load_dict_from_yaml_object_fields(
+            yaml_object=fields_object,
+            object_key="score_name",
+            object_value="label",
+        )
+        renamed_df = df.rename(
+            columns=column_rename_dict,
+            inplace=False,
+        )
 
-        return df_sorted
+        # sort if needed
+        if config_object.get("sort_by_label"):
```

Comment:
again, i get really nervous about strings, can we make this a constant ("sort_by_label")?

Reply:
should `"sort_by_label"` be a constant too?